types/eslint/eslint-tests.ts

```diff
@@ -507,6 +507,7 @@ cli = new CLIEngine({ resolvePluginsRelativeTo: 'test' });
 cli = new CLIEngine({ rules: { 'test/example-rule': 1 } });
 cli = new CLIEngine({ rulePaths: ['foo'] });
 cli = new CLIEngine({ reportUnusedDisableDirectives: true });
+cli = new CLIEngine({ errorOnUnmatchedPattern: false });
```

Comment:
There should be tests also in /ts3.1/eslint-tests.ts

Reply:
@jessetrinity, thank you. I've added it.